projects/plugins/jetpack/_inc/lib/core-api/wpcom-endpoints/class-wpcom-rest-api-v2-endpoint-instagram-gallery.php

```diff
@@ -30,8 +30,8 @@ public function __construct() {
 			}
 		}
 
-		if ( ! class_exists( 'Jetpack_Instagram_Gallery_Helper' ) ) {
-			\jetpack_require_lib( 'class-jetpack-instagram-gallery-helper' );
+		if ( ! class_exists( 'Jetpack_Instagram_Gallery_Helper' ) && defined( 'JETPACK__PLUGIN_DIR' ) ) {
```

Comment:
When would `JETPACK__PLUGIN_DIR` not be defined?

Reply:
Was just going purely on some precedent when poking around other uses 🤷